app/javascript/react/ansibleCatalog/ansibleCatalogItemForm.jsx

```diff
@@ -0,0 +1,375 @@
+                <NavItem eventKey={2}>
+                  {__('Retirement')}
+                </NavItem>
+              </Nav>
+              <TabContent animation>
+                <TabPane eventKey={1} mountOnEnter={true} unmountOnExit={true}>
+                  <AnsiblePlaybookFields
+                    region={region}
+                    prefix='provisioning'
+                    addExtraVars={push}
+                    formValues={values}
+                    changeField={change}
+                  >
+                  <Field
+                    options={dialogOptions}
+                    label={__('Dialog')}
+                    name='dialogOption'
+                    component={FinalFormSelect}
+                  />
+                  <Condition when="dialogOption" is='existing'>
+                    <Field
+                      name='dialog_id'
+                      label={__('Select Dialog')}
+                      validateOnMount= {true}
+                      validate={required({ msg: __('Required') })}
+                      options={dropdowns.dialogs || []}
+                      component={FinalFormSelect}
+                    />
+                  </Condition>
+                  <Condition when="dialogOption" is='new'>
+                    <Field
+                      name='dialog_name'
+                      label={__('New')}
+                      component={FinalFormField}
+                      validateOnMount={true}
+                      validate={value => {
+                        if (!value) {
+                          return (__('Required'));
+                        }
+                        const exists = dropdowns.dialogs.find((dialog) => {
+                          return dialog.label === value;
+                        });
+                        return (exists ? __('Dialog name exists') : '');
+                      }
+                      }
+                    />
+                  </Condition>
+                  </AnsiblePlaybookFields>
+                </TabPane>
+                {/* Retirement tab */}
+                <TabPane eventKey={2} mountOnEnter={true}>
+                  <button type="button" onClick={() => copyValues()}>
+                    {__('Copy from Provisioning')}
+                  </button>
+                  <AnsiblePlaybookFields
+                    region={region}
+                    prefix='retirement'
+                    addExtraVars={push}
+                    formValues={values}
+                    changeField={change}
+                  >
+                  <Field
+                    name='remove_resources'
+                    component={FinalFormSelect}
+                    label={__('Remove Resources?')}
+                    options={getResourceOptions(values)}
+                  />
+                  </AnsiblePlaybookFields>
+                </TabPane>
+              </TabContent>
+            </div>
+          </TabContainer>
+          <FormButtons
```

Comment:
You must also pass callback for the cancel button `cancelClicked`. Otherwise the button won't do anything.

Reply:
added a callback and tested it out.